docs/src/pages/tutorial/run-using-telegram-notification.md

```diff
@@ -20,9 +20,13 @@ Now the next step is to create a group for the Monika notifications. Create a gr
 
 ![](https://miro.medium.com/max/1400/1*jI767UEJfnefsev7JDILOQ.png)
 
-Now that we have our bot API key and set our bot to be a group administrator, all we need to do is to get the group ID. Invite a new bot to the group called **@RawDataBot** into your group to get the group ID.
+Now that we have our bot API key and set our bot to be a group administrator, all we need to do is to get the group ID. Invite one of the following bots into your group to obtain the group ID:
 
-![](https://miro.medium.com/max/762/1*FlZ_WX2zPPqmvpQNY9GWmA.png)
+- [getmyid_bot](https://t.me/getmyid_bot)
+- [getidsbot](https://t.me/getidsbot)
+- [RawDataBot](https://t.me/RawDataBot)
+
```

Comment:
If those bots are not active anymore, you need to find another bot that can obtain the group ID.